install_mac.sh

```diff
@@ -15,12 +20,43 @@ function generate_random_password() {
     echo
 }
 
+function check_file_permissions() {
```

Comment:
Maybe it makes sense to create a shared lib for both install scripts? We are duplicating more and more content in both scripts.

Reply:
yes, will do that in a sepratare pr